larq/models.py

```diff
@@ -79,22 +79,31 @@ def _row_from_stats(stats, summed_stats):
     return (stats.get(key, 0) for key in sorted(summed_stats))
 
 
+def _get_input_precision(layer):
+    try:
+        return layer.input_quantizer.precision
```

Comment:
We could make this more explicit, but I think that's a bit more readable than `getattr(getattr(layer, "input_quantizer", {}), "precision", "?")`